kieserver/image.yaml

```diff
@@ -91,6 +91,12 @@ envs:
     - name: "AUTO_CONFIGURE_EJB_TIMER"
       example: "false"
       description: "Set if the EJB timer datasources should be auto configured, if false, user will need to define the default and the ejb datasources using the DATASOURCES or DB_SERVICE_PREFIX_MAPPING environment variables manually. Default is true"
+    - name: "KIE_EJB_TIMER_TX"
+      example: "false"
+      description: "Set if the EJB timer is in use, if true enables this property enables the transactions for EJB timer. Defaults to true"
```

Comment:
I am not sure what "Set if the EJB timer is in use" means. I would suspect this means that the variable is effective only when the EJB timer is in use? In this case, no need for "Set". I would rewrite thus:
"If the EJB timer is in use, setting this parameter to true enables transactions for EJB timer. The default is true"